card/mod/05_standard/set/all/links.rb

```diff
@@ -31,7 +31,11 @@ def web_link href, opts={}
   # link to a specific card
   def card_link name, opts={}
     text = (opts.delete(:text) || name).to_name.to_show @context_names
-    name = Card[name].cardname if Symbol===name
+    name = case name
+           when Symbol then Card[name].cardname
+           when Card   then name.cardname
+           else             name
+           end
```

Comment:
It seems like it can't hurt to be more specific in how we call it.  If you have the card, adding .cardname isn't ugly, and if you look at the uses, I think cardname is usually what it has already.  I can go either way and this and will defer to Ethan's opinion.

I think the point about looking up the card for the symbol case is more important.  At the very least it should :skip_modules, but we are guaranteed to find the codename in the codenames hash, and that is the fastest lookup we can do here.  We don't need the card here, just the name.